### ISSUE TITLE
First time setup screen hangs in app

### ISSUE THREAD

**Issue by mbz4** [robocarstore/donkeycar-images#7]
Hello, 
having followed instructions available here: 
https://medium.com/robocar-store/robocar-controller-quick-start-guide-bdf8cb16d7ce
... after successfully flashing the latest (pi4_dcv4_v20220602.zip) image to a 16GB SD card, successfully inserting it into the car Pi 4B and connecting to the newly created AP hotspot using Android phone, opening the donkey car app and finding the car listed successfully...
... nothing happens when tapping the donkey car in the list of cars...
Kindly, advise on what could cause this.
In the meantime will try older versions and report back results.

**Comment by sctse999** [comment]
Interesting. If you know the ip of the donkey car, can you head up http://<ip>:8888/vehicle/status and post ther result?

**Comment by mbz4** [comment]
Hey, I'm so sorry for the late reply.
For the prebuilt image in question, the django page displayed, I believe, path & path link (systemctl autostart server...) related errors.
This may be because I the Pi Imager I had used was configured to autoset a hostname & ssh creds prior to first boot. 
This meant that instead of using the built in first config menu in the mobile app, relevant load errors happened., conveniently displayed at the address you provided.
 
Since then, I've gone and setup a new install from scratch which proved useful and a good learning experience. 
Got the app server working by resolving path conflicts and a mishandled link and the first time config to reconfigure the hostname as intended.
In conclusion, I had misinterpreted the resulting behavior due to a SD card imaging related preset. 
I hope you will permit me to close it with this comment.
